tests/end-to-end/director/fixtures_queries_test.go

```diff
@@ -17,12 +17,12 @@ func getApplication(t *testing.T, ctx context.Context, id string) graphql.Applic
 }
 
 func createApplication(t *testing.T, ctx context.Context, name string) graphql.ApplicationExt {
-	in := generateSampleApplicationInputWithName("first", name)
+	in := generateSampleApplicationCreateInputWithName("first", name)
```

Comment:
I wonder if we need function `generateSampleApplicationCreateInputWithName`.
We can just override property after creating the object.